test/test-utils/gameWrapper.ts

```diff
@@ -43,14 +43,15 @@ export class GameWrapper {
     Pokemon.prototype.enableMask = () => null;
     Pokemon.prototype.cry = () => null as any;
     Pokemon.prototype.faintCry = (cb) => {
-      if (cb) cb();
+      if (cb) {
+        cb();
+      }
```

Comment:
if we go with this notation, might as well
```suggestion
      cb?.();
```

Reply:
Yeah that's beter.